doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -113,18 +140,27 @@ term consists of the successive application of its coercions.
 Declaring Coercions
 -------------------------
 
-.. cmd:: Coercion @reference : @class >-> @class
-         Coercion @ident {? @univ_decl } @def_body
+.. cmd:: Coercion @reference {? : @class >-> @class }
+         Coercion @ident_decl @def_body
 
   The first form declares the construction denoted by :token:`reference` as a coercion between
-  the two given classes.  The second form defines :token:`ident`
-  just like :cmd:`Definition` :n:`@ident {? @univ_decl } @def_body`
-  and then declares :token:`ident` as a coercion between it source and its target.
+  the two given classes.  The second form defines :token:`ident_decl`
+  just like :cmd:`Definition` :n:`@ident_decl @def_body`
+  and then declares :token:`ident_decl` as a coercion between it source and its target.
   Both forms support the :attr:`local` attribute, which makes the coercion local to the current section.
 
-  This command supports the :attr:`local` and :attr:`global` attributes
-  (described :ref:`here <set_unset_scope_qualifiers>`) as well as the
-  :attr:`nonuniform` attribute.
+  :n:`{? : @class >-> @class }`
+    The source and target classes of the coercion.
+    If unspecified, :n:`@reference` must already be a coercion, which
+    enables modifying the :attr:`reversible` attribute of :n:`@reference`.
+    See the :ref:`example <example-reversible-coercion-attribute>` below.
+
+  .. attr:: reversible{? = {| yes | no } }
+     :name: reversible
+
+     This :term:`attribute` allows the coercion to be used as a reversible
```

Comment:
Similarly, as suggested by Ali, it would be good to link from here and from the example to the section describing what reversible coercions are.